src/ResourceManager/Profile/Commands.Profile/Account/AddAzureRmAccount.cs

```diff
@@ -163,22 +167,27 @@ public override void ExecuteCmdlet()
 
             AzureAccount azureAccount = new AzureAccount();
 
-            switch(ParameterSetName)
+            if (!string.IsNullOrEmpty(AccessToken))
```

Comment:
I am worried about changing from a switch statement to an if based on the current parameters

Reply:
I'm still seeing the issue where ParameterSetName is null while running these tests. Let's discuss tomorrow.